### ISSUE TITLE
Survey fails to publish form to ArcGIS Portal

### ISSUE THREAD

**Issue by sjblitz** [Esri/Survey123Community#434]
I get the following error when trying to publish a form to ArcGIS portal:

![capture3](https://cloud.githubusercontent.com/assets/16241979/14688768/d6a9398a-0709-11e6-976e-41c06734eb4e.PNG)

I have success publishing this same exact form to ArcGIS Online, however, I am unable to publish it to ArcGIS Portal. Are there some things that ArcGIS Online can handle that ArcGIS Portal can not?

Thanks

**Comment by bowlesgsrcx** [comment]
Are you using Portal 10.4? if not, try shortening the length of the title of your survey "Virtual Trench Inspection" to less than 19 characters in length.

--gary

**Comment by sjblitz** [comment]
No, I am using 10.3. 

I tried shortening the title of my survey to less than 19 characters and I still get the same error, "ERROR: Creating feature class failed."

**Comment by bowlesgsrcx** [comment]
Did you create a new survey with a new name or just change the name of the survey in the "form_title" field on the settings tab? I had this same problem and I had create a new survey form with the shortened title to make them work.

**Comment by sjblitz** [comment]
Yes I created a brand new survey with the shortened title and it still gave me that error. Not sure why I can publish it to ArcGIS Online but not Portal?

**Comment by HannahFerrier** [comment]
Please see the bottom of this page for troubleshooting Portal issues: http://doc.arcgis.com/en/survey123/create-surveys/troubleshoot.htm

**Comment by sjblitz** [comment]
With some help I was able to determine the issue as to why I was unable to Publish my survey to Portal but was able to Publish the same survey to ArcGIS Online. It turns out that my issue was due to the naming convention I used for one of my “Questions”. I was using the Survey123 'Start' and 'End' types in order to capture the start and end times of the survey. In the name field I was using "start" and "end" and this was causing the error. I simply changed the name field to "inspection_start" and "inspection_end" and that seemed to do the trick. I can now publish my survey to Portal. 

Thanks
